build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -5,26 +5,18 @@ steps:
       certSecureFile: 'osx_signing_key.p12'
     condition: eq(variables['signed'], true)
 
-  - script: |
-      mkdir -p .build
-      echo -n $BUILD_SOURCEVERSION > .build/commit
-      echo -n $VSCODE_QUALITY > .build/quality
-    displayName: Prepare cache flag
-
-  - task: 1ESLighthouseEng.PipelineArtifactCaching.RestoreCacheV1.RestoreCache@1
-    displayName: Restore Cache - Compiled Files
+  - task: DownloadPipelineArtifact@2
     inputs:
-      keyfile: 'build/.cachesalt, .build/commit, .build/quality'
-      targetfolder: '.build, out-build, out-vscode-min, out-vscode-reh-min, out-vscode-reh-web-min'
-      vstsFeed: 'BuildCache'
-      platformIndependent: true
-      alias: 'Compilation'
+      artifact: Compilation
+      path: $(Build.ArtifactStagingDirectory)
+    displayName: Download compilation output
+    condition: and(succeeded(), ne(variables['VSCODE_ARCH'], 'universal'))
```

Comment:
Where is this new env var coming from?

Reply:
Got it from vscode. Will change as we don't support multi arch yet.